BrainPortal/app/models/local_data_provider.rb

```diff
@@ -38,6 +38,11 @@ class LocalDataProvider < DataProvider
 
   Revision_info=CbrainFileRevision[__FILE__] #:nodoc:
 
+  # this returns the category of the data provider -- used in view for admins
```

Comment:
Returning a nil is the convention that we'll use to HIDE a data provider class from the interface.
Thus, returning a NIL